### PR TITLE
Add new code examples for bidding data exclusion and seasonality adju…

### PR DESCRIPTION
**Pull request by AnashOommen**
…stments

### CODE REVIEW

File: examples/AdvancedOperations/AddBiddingDataExclusion.cs [hunk 1]
```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 Google LLC
```
Comment (christopherseeley) [inline examples/AdvancedOperations/AddBiddingDataExclusion.cs:1]: 2021

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs [hunk 2]
```diff
@@ -0,0 +1,192 @@
+// Copyright 2020 Google LLC
```
Comment (christopherseeley) [inline examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs:1]: 2021

**Review by christopherseeley — APPROVED** [review]

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+            }
+            catch (GoogleAdsException e)
+            {
+                Console.WriteLine("Failure:");
+                Console.WriteLine($"Message: {e.Message}");
+                Console.WriteLine($"Failure: {e.Failure}");
+                Console.WriteLine($"Request ID: {e.RequestId}");
+                throw;
+            }
+
```
Comment (AndrewMBurke) [inline examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs:188]: Nit: extra line.

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+                    // The start time of the seasonality adjustment (in
+                    // yyyy-MM-dd HH:mm:ss format) in the account's timezone.")]
+                    options.StartDateTime = "INSERT_START_DATE_TIME_HERE";
+
+                    // The end time of the seasonality adjustment (in
+                    // yyyy-MM-dd HH:mm:ss format) in the account's timezone.")]
+                    options.EndDateTime = "INSERT_END_DATE_TIME_HERE";
+
+                    options.ConversionRateModifier =
```
Comment (AndrewMBurke) [inline examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs:101]: Missing explanatory comment for this argument.

**Review by AndrewMBurke — APPROVED** [review]
Two minor nits, otherwise LGTM.

File: examples/AdvancedOperations/AddBiddingDataExclusion.cs (hunk 1 above)
Comment (AnashOommen) [reply examples/AdvancedOperations/AddBiddingDataExclusion.cs]: Done.

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs (hunk 2 above)
Comment (AnashOommen) [reply examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs]: Done

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs (hunk 3 above)
Comment (AnashOommen) [reply examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs]: Done.

File: examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs (hunk 4 above)
Comment (AnashOommen) [reply examples/AdvancedOperations/AddBiddingSeasonalityAdjustment.cs:103]: Done.

**Review by AndrewMBurke — APPROVED** [review]

**Review by christopherseeley — APPROVED** [review]

**Pull request merged**